lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -405,6 +405,22 @@ _.assign(Builders.prototype, {
         });
     },
 
+    /**
+     * Creates an ordering field for responses of a V2 Item.
+     *
+     * @param {Object} item - The v2 item to extract saved responses from.
+     * @returns {Object[]} - The order of responses within the V2 Item.
+     */
+    responses_order: function (item) {
+        var allResponses = item.responses || item.response;
+
+        if (!allResponses) {
+            return [];
```

Comment:
Should we check for `retainEmptyValues` here? 🤔

Reply:
We're replicating the same behavior as expected from `folders_order` and `order` fields. I'm of the opinion that we do not need to check `retainEmptyValues` here.